v20.2/alter-type.md

```diff
@@ -0,0 +1,86 @@
+---
+title: ALTER TYPE
+summary: The ALTER TYPE statement modifies a user-defined data type in a database.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> The `ALTER TYPE` [statement](sql-statements.html) modifies a user-defined, [enumerated data type](enum.html) in a database.
+
+## Synopsis
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/alter_type.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------|------------
+`type_name` | The name of the user-defined type.
+`ADD VALUE` | Add a constant value to the user-defined type's list of values.
```

Comment:
make sure the cover the `ADD VALUE {BEFORE | AFTER} 'val'` option. This adds a new value to the enum, but places its sort order relative to the target existing value.